src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -111,222 +57,127 @@ export const TransactionGenericRow = ({
   status,
   title,
   transfersByType,
-  fee,
-  txid,
   txData,
-  blockTime,
-  txLink,
-  compactMode = false,
   toggleOpen,
-  parentWidth,
 }: TransactionGenericRowProps) => {
-  const {
-    columns,
-    dateFormat,
-    breakPoints: [isLargerThanLg],
-  } = GetTxLayoutFormats({ parentWidth })
-
   const txMetadataWithAssetId = useMemo(() => getTxMetadataWithAssetId(txData), [txData])
 
-  const transfers = useMemo(() => {
-    return Object.values(transfersByType).map((transfersOfType, index) => {
-      const hasManyTypeTransfers = transfersOfType.length > 1
-
+  const renderSendInfo = useMemo(() => {
+    if (transfersByType.Send && transfersByType.Send.length > 1) {
```

Comment:
@reallybeard reading this again, this doesn't look out of place, though we may want to keep this encapsulated in utils the same as `transfersOfType` previously